src/content/docs/en/guides/cms/statamic.mdx

```diff
@@ -0,0 +1,102 @@
+2. **A Statamic site** - [Enable REST API](https://statamic.dev/rest-api#enable-the-api) or [GraphQL API](https://statamic.dev/graphql#enable-graphql) by adding `STATAMIC_API_ENABLED=true` or `STATAMIC_GRAPHQL_ENABLED=true` in the `.env` file and enable required resources in the API configuration file.
+
+:::note
+All the examples assume that your website has a collection called `posts`, that has a blueprint called `post`, and this blueprint has a title field (fieldtype text) and content (fieldtype markdown).
+:::
+
+### Setting up Credentials
+
+At the moment, Statamic doesn't provide any native authentication methods.
+
+### Installing Dependencies
+
+There are no dependencies required to fetch your Statamic data from Astro via the REST API.
+
+### Fetching Data
+
+#### REST API
+
+Fetch your Statamic data from your site's REST API URL. By default, it's `https://[YOUR-SITE]/api/`. Then, you can render your data properties using Astro's `set:html={}` directive. 
+
+For example, to display a list of titles and their content from a selected [collection](https://statamic.dev/collections):
+
+```astro title="src/pages/index.astro
+---
+const res = await fetch("https://[YOUR-SITE]/api/collections/posts/entries?sort=-date")
+const posts = await res.json()
+---
+<h1>Astro + Statamic 🚀</h1>
+{
+  posts.map((post) => (
+      <h2 set:html={post.title} />
+      <p set:html={post.content} />
+  ))
+}
+```
+
+#### GraphQL
+
+Fetch your Statamic data with your site's GraphQL API URL. By default, it's `https://[YOUR-SITE]/graphql/`. Then, you can render your data properties using Astro's `set:html={}` directive. 
+
+For example, to display a list of titles and their content from a selected [collection](https://statamic.dev/collections):
+
+```astro title="src/pages/index.astro
+---
+const res = await fetch("https://[YOUR-SITE]/graphql/",
+  {
+    method: 'POST',
+    headers: {'Content-Type':'application/json'},
+    body: JSON.stringify({
+      query: `
+        posts: entries(collection: "posts", sort: "date desc") {
+          data {
+            title
+            ... on Entry_Posts_Post {
+              content
+            }
+          }
+        }
+      `
+    }),
+  });
+const entries = await res.json()
+---
+<h1>Astro + Statamic 🚀</h1>
+{
+  entires.data.posts.data.map((post) => (
+      <h2 set:html={post.title} />
+      <p set:html={post.content} />
+  ))
+}
+```
+
+### Publishing your site
+
+To deploy your site visit our [deployment guide](/en/guides/deploy/) and follow the instructions for your preferred hosting provider.
+
+## Community Resources 
+
+- [How to build a static site using Statamic as headless CMS](https://buddy.works/guides/statamic-rest-api)
+- [Implementing Astro live previews in headless Statamic](https://maciekpalmowski.dev/implementing-live-previews-in-headless-statamic-when-using-astro/)
```

Comment:
Cool of you to include these!

Reply:
I'm planning to work on repos too in the upcoming time :)